src/main/java/org/websoso/WSSServer/repository/UserNovelCustomRepositoryImpl.java

```diff
@@ -73,4 +59,40 @@ public UserNovelCountGetResponse findUserNovelStatistics(User user) {
                 .where(userNovel.user.eq(user))
                 .fetchOne();
     }
+
+    @Override
+    public List<Long> findTodayPopularNovelsId(Pageable pageable) {
+        LocalDate sevenDaysAgo = LocalDate.now().minusDays(7);
+
+        return jpaQueryFactory
+                .select(userNovel.novel.novelId)
+                .from(userNovel)
+                .where(userNovel.status.eq(ReadStatus.WATCHING)
+                        .or(userNovel.status.eq(ReadStatus.WATCHED))
+                        .or(userNovel.isInterest.isTrue())
+                        .and(userNovel.createdDate.after(sevenDaysAgo.atStartOfDay())))
```

Comment:
p2; 선생님 뭔가 이게 맞는 것 같은데 저도 헷갈려서 한번만 확인 부탁드림니다...
```suggestion
                .where(userNovel.status.eq(ReadStatus.WATCHING)
                        .or(userNovel.status.eq(ReadStatus.WATCHED))
                        .or(userNovel.isInterest.isTrue()))
                        .and(userNovel.createdDate.after(sevenDaysAgo.atStartOfDay()))
```

Reply:
무조건입니다 ㅋㅋㅋㅋㅋ 와우 신의 눈

감사하니다

Reply:
```java
    .where((userNovel.status.eq(ReadStatus.WATCHING)
        .or(userNovel.status.eq(ReadStatus.WATCHED))
        .or(userNovel.isInterest.isTrue()))
        .and(userNovel.createdDate.after(sevenDaysAgo.atStartOfDay())))
```
where 안에 (), ()이 들어가야 해서 where바로 뒤에 `(` 하나 추가했습니닷